src/Core/Application/Dto/TaskYaml/TaskYamlInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Dto\TaskYaml;
+
+interface TaskYamlInterface
```

Comment:
According to current implementation not necessary to have an interface for dto. Please remove.

Reply:
Fixed